e2e_tests/helpers.go

```diff
@@ -12,3 +14,11 @@ func RequireCurrentBranchName(t *testing.T, repo *git.Repo, name string) {
 	require.NoError(t, err, "failed to determine current branch name")
 	require.Equal(t, name, currentBranch, "expected current branch to be %q, got %q", name, currentBranch)
 }
+
+func GetStoredParentBranchState(t *testing.T, repo *git.Repo, name string) meta.BranchState {
+	// We shouldn't do this as part of an E2E test, but it's hard to ensure otherwise.
+	db, err := jsonfiledb.OpenRepo(repo)
```

Comment:
Do we need to close this to close the file handle? (not sure if that's actually a part of the DB interface in hindsight... 🤔)

Reply:
jsonfiledb doesn't have a file handle in it.

Reply:
Now that you say that I remember it lol. I wrote that code all of two weeks ago and remember none of it apparently.